metadata-etl/src/main/resources/jython/HdfsLoad.py

```diff
@@ -61,7 +61,7 @@ def load_metadata(self):
         update stg_dict_dataset
         set name = substring_index(urn, '/', -2)
         where db_id = {db_id}
-          and name in ('1.0', '2.0', '3.0', '4.0', '0.1', '0.2', '0.3', '0.4', 'dedup', '1-day', '7-day');
+          and name '[0-9]+\\.[0-9]+|dedup|dedupe|[0-9]+-day';
```

Comment:
Does this works? I think there should be a "regexp" between name and the pattern.